classes/metric/online_users_metric.php

```diff
@@ -37,4 +49,99 @@ class online_users_metric extends builtin_user_base {
     public function get_name(): string {
         return 'onlineusers';
     }
+
+    /**
+     * The metric type.
+     *
+     * @return int
+     */
+    public function get_type(): int {
```

Comment:
Is this needed in this patch? Seems unrelated and isn't used elsewhere in the patch?

Reply:
No it is present in parent class, removed it.